src/pages/Admin/BackgroundSelect/BackgroundSelect.tsx

```diff
@@ -59,16 +60,26 @@ export const BackgroundSelect: React.FC<BackgroundSelectProps> = ({
     [isUploading, mapBackgrounds, uploadMapBackground]
   );
 
+  const onClick = useCallback(
+    ({ url, files }: FileButtonOnChangeData) => uploadMapBackground(url, files),
+    [uploadMapBackground]
+  );
+
   return (
     <div className="BackgroundSelect">
       {!mapBackground && (
         <>
           <FileButton
             disabled={isUploading}
             title="Import a map background"
-            description="Recommended size: 2000px / 1200px"
-            onChange={uploadMapBackground}
-          />
+            variant="primary"
+            onClick={onClick}
+          >
+            Import a map background
+          </FileButton>
+          <span className="BackgroundSelect__file-description">
+            Recommended size: 2000px / 1200px
+          </span>
```

Comment:
NOTE: Extracted from inside `FileButton`. Getting the description tied down within the file `FileButton` component limits the options for its placement as well as reusability of the button in other contexts.